### ISSUE TITLE
the string library might be more relevant for 06-libraries

### ISSUE THREAD

**Issue by elliewix** [data-lessons/library-python-intro#29]
Episode 6 on libraries and modules uses the math library.  The examples could be pretty easily rewritten using the string library.  Example:

```
import string
print("the lower ascii letters are", string.ascii_lowercase)
```
I can work on a PR for this if there's approval for the change.

The challenges will also need to be changed to be less math/genomics focused.  For example, the Jugsaw Puzzle to fix the code generating a random DNA base pare wouldn't be very relatable.

**Comment by jt14den** [comment]
+1 from me. I think the string library fits better with what librarians do/have.

**Comment by elliewix** [comment]
OK, I can take this one on.  I tend to use the string module a lot, so hopefully I can come up with some neat examples.

**Comment by c-martinez** [comment]
Closed via #56
